src/Sylius/Bundle/PromotionBundle/Resources/translations/messages.en.yml

```diff
@@ -32,6 +37,15 @@ sylius:
         promotion_filter:
             price_range: Price range filter
         promotion_rule:
+            cart_quantity: Cart quantity
+            contains_product: Contains product
+            item_total: Item total
+            nth_order: Nth order
+            shipping_country: Shipping country
+            taxonomy: Taxonomy
+            customer_group: Customer group
+            has_at_least_one_from_taxons: Has at last one from taxons
```

Comment:
And there is a typo here - it should "at l**e**ast". And that's why the build is failing.